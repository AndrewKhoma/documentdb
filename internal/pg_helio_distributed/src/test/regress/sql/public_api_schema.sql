-- show all functions exported in helio_api_distributed: This should stay empty
\df helio_api_distributed.*
