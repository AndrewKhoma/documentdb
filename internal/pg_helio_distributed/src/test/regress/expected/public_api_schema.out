-- show all functions exported in helio_api_distributed
\df helio_api_distributed.*
                                     List of functions
        Schema         |        Name        | Result data type | Argument data types | Type 
-----------------------+--------------------+------------------+---------------------+------
 helio_api_distributed | complete_upgrade   | boolean          |                     | func
 helio_api_distributed | initialize_cluster | void             |                     | func
(2 rows)

-- show all aggregates exported
\da+ helio_api_distributed.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
--------+------+------------------+---------------------+-------------
(0 rows)

-- This is last (Tables/indexes)
\d helio_api_distributed.*
      Table "helio_api_distributed.helio_cluster_data"
  Column  |      Type       | Collation | Nullable | Default 
----------+-----------------+-----------+----------+---------
 metadata | helio_core.bson |           |          | 

